.github/workflows/esc-cli.yml

```
@@ -0,0 +1,95 @@
+env:
+  GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
+
+name: esc cli docs build
+on:
+  repository_dispatch:
+    types:
+      - esc-cli
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    needs: build-esc-cli-docs
+    steps:
+      - name: checkout docs repo
+        uses: actions/checkout@v2
+      - name: set the esc version
+        run: |
+          echo "ESC_VERSION=${{ github.event.client_payload.ref }}" >> $GITHUB_ENV
+      - name: pull-request
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: "pulumi/${{ github.run_id }}-${{ github.run_number }}"
+          destination_branch: "master"
+          pr_title: "Regen docs esc@${{ env.PULUMI_VERSION }}"
```

should this be `ESC_VERSION`?

```suggestion
          pr_title: "Regen docs esc@${{ env.ESC_VERSION }}"
```

---

Yes, good catch